README.md

```diff
@@ -1,21 +1,23 @@
 # osu! wiki [![dev chat](https://discordapp.com/api/guilds/188630481301012481/widget.png?style=shield)](https://discord.gg/ppy)
 
-The home of [osu!](https://osu.ppy.sh) wiki.
+<!--
+Uncomment and replace after the main wiki page gets implemented, link isn't official:
+Home of the [Knowledge Base](https://osu.ppy.sh/help/wiki).
+-->
+
+Home of the [osu!](https://osu.ppy.sh) wiki.
 
 ## Requirements
 
-1. A [GitHub account](https://help.github.com/articles/signing-up-for-a-new-github-account/ "GitHub Help in Signing up an account").
-2. A general understanding of the feature-branch workflow.
-3. Knowledge in [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet "adam-p's cheatsheet for Markdown"), specifically [GitHub-Flavoured Markdown(GFM)](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/ "GitHub Help on Writing on GitHub").
-4. Ability to write in English, preferably British English.
+1. General understanding of the feature-branch workflow (i.e. pull request a branch that is not "master").
+2. Knowledge in [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet "adam-p's cheatsheet for Markdown"), specifically [GitHub-Flavoured Markdown(GFM)](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/ "GitHub Help: Writing on GitHub").
+3. Ability to write in English, preferably British English.
 
 ## Getting Started
```

Comment:
I think you should either change this to a "helpful links to get started" section and make the contents a list or, alternatively, change these sentences to be grammatically complete. As is, the header hints at a guide of sorts, but you just list some links instead.

Reply:
I'll just make a sub header called `Helpful Links` and list them, since the title you mention is too long.